### DISCUSSION TITLE
no Firefly login possible after Linux Ubuntu installation

### DISCUSSION THREAD

**Discussion by Ostseejosh** [iotaledger/firefly#3045] [Help & Support] [answered]
Ubuntu Installation was established on my new computer by copying all programs from the former computer.
Everything is fine but Firefly.
On the former computer Firefly is running with 2 wallets / ASMB und SMR staked / everything is fine!
On the new Computer the Firefly PIN is not accepted. I can't login.
I tried to create a new profil:
- I have a Ledger-Buckup
- I have a Firefly Ledger-Backup
- I specified a PIN / I confirmed the PIN
- Connect my Ledger to Firefly
- I unlocked my Ledger and opened the IOTA App

=> no connection to Firefly was built up (see attached file)
![Firefly_connection](https://user-images.githubusercontent.com/104463432/165389023-85012bba-5986-484d-acc5-eaebbffc8082.png)

I tried to delete Firefly and install Firefly 1.5.1 again -> this doesn't help -> same problem again
Maybe I need to fully erase Firefly from the Ubuntu platform including the login information (Firefly Profil + PIN)  
-> I don't know how?? I tried but I failed.

What can I do?
I needed to copy everything from the former Computer (including Firefly). It's no option for me to start again from a clean Ubuntu installation. 

Thank You!!
Ostseejosh

views.dashboard.security.version.title
speech: de
currency: USD
Node-selection: automatic
Plattform: linux
Plattformversion: 5.13.0-40-generic
Plattform-Architectur: x64
CPUs: 4
storage: 7699.2 MB
free storage: 2928.8 MB

**Comment by begonaalvarezd** [comment]
Hi, does your ledger successfully connect to the official Ledger Live App? 

If so, please make sure that Ledger Live is not connected while Firefly is open as Ledger can only be connected to one app at a time.

On the other hand, if Ledger Live also doesnt detect your Ledger device, you might be missing their udev rules in your new machine which are needed for the ledger to be detected by your system. Please refer to their original troubleshoot to try fixing your connection issue: ["Fix USB connection issues with Ledger Live"](https://support.ledger.com/hc/en-us/articles/115005165269-Fix-USB-connection-issues-with-Ledger-Live?support=true).

**Reply by Ostseejosh** [reply]
Thank You for your answer!!
Yes, my ledger successfully connects to the official Ledger Live App.
There was only single use when I tried the Firefly connection on my new Ubuntu computer.
When I installed Firyfly 1.5.1 on a Windows 10 Laptop everything is fine. On Windows I have Ledger connection with Firefly. Also on my old Ubuntu Computer.
Probably it is a Problem with the new Ubuntu Installation. Maybe I should remove Firefly entirly from Ubuntu and reinstall.
Therefore I try to find help at Ubuntu support...
Or do You have more ideas??

Thanks, Ostseejosh

**Reply by Ostseejosh** [reply]
The Ubuntu Community couldn't help me. 
I still have the problem and can not use Firefly one my new Ubuntu computer :-(

**Comment by MiKeF05** [comment]
You should uninstall Firefly completely. To do this uninstall Firefly and remove the folder ~/.config/Firefly. This also removes all your profiles. So make sure to have a backup to restore your profiles.

**Comment by Ostseejosh** [comment]
thank You very much, MiKeF05!
Now I have a initial Firefly without profil. 
I did not know ~/.config/Firefly. Thank You!
That was the easy part. I still don't have connection. Probably I need udev rules (as  begonaalvarezd wrote).
I will check ["Fix USB connection issues with Ledger Live"](https://support.ledger.com/hc/en-us/articles/115005165269-Fix-USB-connection-issues-with-Ledger-Live?support=true).

**Comment by Ostseejosh** [comment, marked as answer]
okay, I set up udev rules.
And YES ! I do have connection to the Ledger.
And I have my Firefly wallets with IOTA, ASMB, SMR on my new computer.
STRIKE!
Thank You!
